README.md

```diff
@@ -141,6 +142,7 @@ ever.
 
 #### Assembly
 * [Enough x86 Assembly to Be Dangerous](https://www.youtube.com/watch?v=IfUPkUAEwrk) [00:30:58] by **Charles Bailey** (2017)
+* [The Teensy ELF Executable](https://www.muppetlabs.com/~breadbox/software/tiny/techtalk.html) [00:49:04] by **Brian Raiter** (2014)
```

Comment:
This talk is `[00:49:03]` long for me. Apparently our computers disagree slightly on video lengths! (I don’t really mind, I just thought it was interesting)

Reply:
this wasn't on Youtube of course, so i had to download it and I looked at the timeline in GNOME Totem which says `-49:04`

VLC says 49:03 too, and Shotcut (an editor) says it's 49:03 + 1 frame, so Totem is rounding up. We'll round down though to agree with most applications, I'll change it